class/practice2.1/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+
+void loadStatistics(std::unordered_map<std::string, int>& stats) {
+    std::ifstream infile("statistics.txt");
+    std::string name;
+    int count;
+    while (infile >> name >> count) {
+        stats[name] = count;
+    }
+}
+
+void saveStatistics(const std::unordered_map<std::string, int>& stats) {
+    std::ofstream outfile("statistics.txt");
+    for (const auto& entry : stats) {
+        outfile << entry.first << " " << entry.second << std::endl;
+    }
+}
+
+int main(int argc, char* argv[]) {
+    if (argc < 2 || argc > 3) {
+        std::cerr << "Usage: " << argv[0] << " <name> [delete]" << std::endl;
+        return 1;
+    }
+
+    std::string name = argv[1];
+
+    if (argc == 3 && std::string(argv[2]) != "delete") {
```

Comment:
use named constants instead of hardcoded lines.
also it seems that you can parse commands at the start like
string name = ...;
bool deleteFlag = ...;
and then you will not need argc argv anymore and can rely on that 2 variables.